src/index.html

```diff
@@ -9,6 +9,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
```

Comment:
```suggestion
<link rel="preconnect" href="https://fonts.googleapis.com">
    <link
      rel="preconnect"
      href="https://fonts.gstatic.com"
      crossorigin="anonymous"
    >
    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
  </head>
```
its better to add font import to html

Reply:
what is means ="anonymous" could you explain ?
![image](https://user-images.githubusercontent.com/70288388/194414264-f7241d48-19ee-4177-9151-22e837943a6e.png)
i don't see it there 
![image](https://user-images.githubusercontent.com/70288388/194414521-3742a72b-1000-4f32-8b8f-eeb119bd4483.png)

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/crossorigin
![image](https://user-images.githubusercontent.com/99331793/195569284-37538c66-7c56-4a4b-9da4-9de993bf2a30.png)